wikimedia/wikipedia-ios
# Image recommendations - Add pt-br translations

**Phabricator:** https://phabricator.wikimedia.org/T364156

### Notes
Adds translations for pt-br.

### Test Steps
1. Set device language to PT-BR
2. Fresh install app.
3. Confirm image recommendations strings are localized.
